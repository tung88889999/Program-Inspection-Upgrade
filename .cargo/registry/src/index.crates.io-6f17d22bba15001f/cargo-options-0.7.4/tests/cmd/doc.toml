bin.name = "cargo-mimic"
args = "doc --help"
