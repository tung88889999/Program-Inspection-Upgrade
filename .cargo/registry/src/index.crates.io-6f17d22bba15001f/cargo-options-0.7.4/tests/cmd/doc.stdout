Build a package's documentation

Usage: cargo-mimic[EXE] doc [OPTIONS]

Options:
  -q, --quiet                   Do not print cargo log messages
      --message-format <FMT>    Error format
  -v, --verbose...              Use verbose output (-vv very verbose/build.rs output)
      --color <WHEN>            Coloring: auto, always, never
      --config <KEY=VALUE>      Override a configuration value (unstable)
  -Z <FLAG>                     Unstable (nightly-only) flags to Cargo, see 'cargo -Z help' for
                                details
      --no-deps                 Don't build documentation for dependencies
      --document-private-items  Document private items
      --open                    Opens the docs in a browser after the operation
      --ignore-rust-version     Ignore `rust-version` specification in packages
  -h, --help                    Print help

Compilation Options:
  -j, --jobs <N>                Number of parallel jobs, defaults to # of CPUs
      --keep-going              Do not abort the build as soon as there is an error (unstable)
      --profile <PROFILE-NAME>  Build artifacts with the specified Cargo profile
      --target <TRIPLE>         Build for the target triple [env: CARGO_BUILD_TARGET=]
      --target-dir <DIRECTORY>  Directory for all generated artifacts
      --timings[=<FMTS>...]     Timing output formats (unstable) (comma separated): html, json
  -r, --release                 Build artifacts in release mode, with optimizations
      --unit-graph              Output build graph in JSON (unstable)

Feature Selection:
  -F, --features <FEATURES>  Space or comma separated list of features to activate
      --all-features         Activate all available features
      --no-default-features  Do not activate the `default` feature

Manifest Options:
      --frozen                Require Cargo.lock and cache are up to date
      --locked                Require Cargo.lock is up to date
      --offline               Run without accessing the network
      --manifest-path <PATH>  Path to Cargo.toml

Package Selection:
  -p, --package [<SPEC>]  Package to document
      --workspace         Document all packages in the workspace
      --exclude <SPEC>    Exclude packages from the build
      --all               Alias for --workspace (deprecated)

Target Selection:
      --lib               Document only this package's library
      --bin [<NAME>]      Document only the specified binary
      --bins              Document all binaries
      --example [<NAME>]  Document only the specified example
      --examples          Document all examples

Run `cargo help doc` for more detailed information.
